backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAO.java

```diff
@@ -457,7 +457,7 @@ void logDatasets(String experimentRunId, List<Artifact> datasets, boolean overwr
   void logVersionedInput(LogVersionedInput request)
       throws InvalidProtocolBufferException, ModelDBException, NoSuchAlgorithmException;
 
-  void deleteLogVersionedInputs(Session session, Long repoId, String commitHash);
```

Comment:
why was the removal necessary?

Reply:
It's not necessary. It's just not used anymore. Reverted it